server/plugin/subscriptions.go

```diff
@@ -15,24 +16,39 @@ import (
 const (
 	SubscriptionsKey     = "subscriptions"
 	excludeOrgMemberFlag = "exclude-org-member"
+	renderStyleFlag      = "render-style"
```

Comment:
We could enrich the flags to also include supported values, instead of only the name. Then we could have a generic validation function lower down in `AddFlag`.

Reply:
This is likely out of scope, since we would want to apply this to all flags potentially. It would be a great addition if you're looking to implement this separately 🙂

Reply:
@TeodorPt I meant to mention you in the comment above

Reply:
@mickmister sure, I'll take that as a separate thing, but it might take a while as my schedule is quite full. In the meantime, this PR is ready for a re-review :)